From c9885536b6b98ae1d60552ab672e7c8cac3f3f7c Mon Sep 17 00:00:00 2001
From: Wu Zhangjin <wuzhangjin@gmail.com>
Date: Sun, 5 May 2019 06:04:53 -0700
Subject: [PATCH 1/2] virt: arm64: Kconfig

disable GCC_PLUGIN support by default.

this option can not be configured in defconfig, whenever enable or disable,
therefore it breaks the auto configuration logic, so, for it is not used for
our requirement, just remove it.

Signed-off-by: Wu Zhangjin <wuzhangjin@gmail.com>
---
 arch/arm64/Kconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index a4168d3..358e043 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -136,7 +136,6 @@ config ARM64
 	select HAVE_FTRACE_MCOUNT_RECORD
 	select HAVE_FUNCTION_TRACER
 	select HAVE_FUNCTION_GRAPH_TRACER
-	select HAVE_GCC_PLUGINS
 	select HAVE_GENERIC_DMA_COHERENT
 	select HAVE_HW_BREAKPOINT if PERF_EVENTS
 	select HAVE_IRQ_TIME_ACCOUNTING
-- 
2.7.4

